code/datums/components/crafting/recipes.dm

```diff
@@ -23,3 +23,12 @@
   */
 /datum/crafting_recipe/proc/check_requirements(mob/user, list/collected_requirements)
 	return TRUE
+
+/datum/crafting_recipe/gripperoffbrand
```

Comment:
I don't think that's a good idea, we have no sticky tape and fingerless gloves are easy to get by.

Reply:
Ill port the sticky tape i suppose

Reply:
Just messed around with the recipy, see if its sufficent